src/service/platform-impl.js

```diff
@@ -230,8 +230,8 @@ export class Platform {
 
 /**
  * @param {!Window} window
- * @return {*} TODO(#23582): Specify return type
+ * @return {undefined}
```

Comment:
Just remove `@return` entirely?

Reply:
That'll work! Just checked and the `jsdoc/require-returns` rule only requires a return if the function actually returns something.